python/cudf/cudf/core/index.py

```diff
@@ -79,6 +79,18 @@ def serialize(self):
     def __contains__(self, item):
         return item in self._values
 
+    def get_level_values(self, level):
+        if level == self.name:
+            return cudf.Series(self)
```

Comment:
Is `Series` the appropriate type to return here?

Reply:
Nope - it's an Index. Fixed.